src/index.html

```diff
@@ -7,7 +7,60 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars--0 stars"> 
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+    
+    
+      <div class="stars stars--1">
+        <div class="stars__activestar"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+          
+      <div class="stars stars--2">
+        <div class="stars__activestar"></div>
+        <div class="stars__activestar"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--2">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
У кожної зірки має бути лише клас stars__star.
А backgrounf-image має визначатись за допомогою модифікатора контейнера.
Тобто, якщо в контейнера клас "stars stars--2", то в css потрібно прописати селектор, який буде визначати, скільки зірок всередині мають мати star-active на бекграунді.

Для цього можна використовувати комбінацію селекторів
`.stars--2 :nth-child(-n+2)`